inc/ds-nagioslivestatus.inc.php

```diff
@@ -93,8 +93,8 @@ static function get_host($hostname) {
         # Split rows into columns and munge into new array
         $hosts = Array();
         foreach($data as $d) {
-            $e = explode("`", $d);
-            $hosts[$e[$cols["display_name"]]] = $e;
+            $explode = explode("`", $d);
+            $hosts[$explode[$cols["display_name"]]] = $explode;
```

Comment:
`$explode` being the same as the function name seems like a bad idea, how about `$exploded`?